backtrace-library/src/main/java/backtraceio/library/common/DeviceAttributesHelper.java

```diff
@@ -187,20 +186,23 @@ private WifiStatus getWifiStatus() {
      * @return is power saving mode activated
      */
     // TODO: replace bool to enum
+    /*
```

Comment:
Why do we remove this? I also don't see this power saving mode check re-implemented anywhere else?